lib/url_api.ex

```diff
@@ -0,0 +1,26 @@
+alias GBitly.Repo
+
+defmodule GBitly.UrlApi do
+  def verify_if_exists_url(from) do
+    case Repo.query("SELECT COUNT(*) AS total FROM urls WHERE url_name LIKE '#{from}'") do
```

Comment:
Aqui tem a função Repo.exists? que retorna um booleano
```suggestion
    case Repo.exists?(from u in Url, where: u.url_name like(^from)) do
```